service/telemetry/logger.go

```diff
@@ -28,9 +32,40 @@ func newLogger(cfg LogsConfig, options []zap.Option) (*zap.Logger, error) {
 	}
 
 	logger, err := zapCfg.Build(options...)
+
 	if err != nil {
 		return nil, err
 	}
+
+	if len(cfg.Processors) > 0 {
+		sdk, err := config.NewSDK(
+			config.WithContext(ctx),
+			config.WithOpenTelemetryConfiguration(
+				config.OpenTelemetryConfiguration{
+					LoggerProvider: &config.LoggerProvider{
+						Processors: cfg.Processors,
+					},
+					// TODO: add resource
+					// Resource: &config.Resource{},
+				},
+			),
+		)
+
+		if err != nil {
+			return nil, err
+		}
+
+		logger = logger.WithOptions(zap.WrapCore(func(c zapcore.Core) zapcore.Core {
+			return zapcore.NewTee(
+				logger.Core(),
```

Comment:
The linting error made me think you can just do this
```suggestion
				c,
```

Reply:
fixed